basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -1462,14 +1450,11 @@ public static DeepLinkResponse getDeepLinkFromToken(Map<String, Object> tool, St
 		Long toolVersion = getLongNull(tool.get(LTIService.LTI_VERSION));
 		boolean isLTI1 = toolVersion == null || (!toolVersion.equals(LTIService.LTI_VERSION_2));
 		boolean isLTI2 = !isLTI1;  // In case there is an LTI 3
+		boolean isLTI13 = isLTI13(tool, null);
 
-		// LTI 1.3 is a variation on  LTI 1.1
-		Long toolLTI13 = getLongNull(tool.get(LTIService.LTI13));
-		boolean isLTI13 = Objects.equals(toolLTI13, 1L);
-		if (secret == null || consumerkey == null && Objects.equals(toolLTI13, 1L)) {
-			isLTI13 = true;  // No way to launch LTI 1.1
-		}
 		log.debug("toolVersion={} isLTI1={} isLTI13={}", toolVersion, isLTI1, isLTI13);
+		System.out.println("2 toolVersion={} isLTI1={} isLTI13={}"+toolVersion+isLTI1+isLTI13);
```

Comment:
log instead of System.out

Reply:
Got one - missed one.  Glad my log statements are working again - I won't have to add these and then forget to take them out later.